packages/frontend/src/components/Layout/index.tsx

```diff
@@ -61,21 +76,24 @@ export default function Layout({
     <>
       <SiteWideBanner />
       <RestrictedGovtMasthead />
-      <AppBar
-        drawerOpen={isDrawerOpen}
-        onDrawerOpen={openDrawer}
-        onDrawerClose={closeDrawer}
-      />
-      <Box sx={{ display: 'flex', flex: 1 }}>
-        <Drawer
-          links={drawerLinks}
-          open={isDrawerOpen}
-          onOpen={openDrawer}
-          onClose={closeDrawer}
-        />
+      <AppBar />
+      <LayoutNavigationProvider value={layoutNavigationProviderData}>
+        <Box sx={{ display: 'flex', flex: 1 }}>
+          <Show above="sm">
+            <Box mt={1}>
+              <NavigationSidebar />
+            </Box>
+            <Box>
+              <Divider
```

Comment:
nit: very tempted to see if we can replace this with a `borderEnd` styling on `Box` around `NavigationSidebar` / even inside `NavigationSidebar`, but a bit too late now to debug why that doesn't work.

Reply:
tbh i could just add a `borderRight` to the `Box` with the `NavigationSidebar` but I felt that the `Divider` was a more semantically correct manner to show the divider component